### ISSUE TITLE
Add include option for batch to include extra directories in its search

### ISSUE THREAD

**Issue by miniscruff** [miniscruff/changie#249]
`--include` option which allows adding other directories to the batch command, in addition to unreleased.

Example: 
```
.changes
  unreleased
  betas
  candidates
```

`changie batch minor --include betas --include candidates` or 
`changie batch minor --include betas,candidates`

Few rules:
1. If an included folder does not exist, changie should throw an error
2. If an included folder is empty, no changes are added and it is basically just skipped ( this allows you keep a set of folders around )
3. The folder is not to be deleted or edited
4. Files are to be removed from the included folders, unless `--keep` is also included
5. `--move` should move all files from all included folders into the one passed in ( not sure on the use case here but not allowing move or skipping it feels wrong )
6. Configured move directory can be part of included, in which case those files would stay in place ( not sure the use case here either )
7. No configuration is required as include would only be used when releasing a final version of many others and would not work any other time

Part of https://github.com/miniscruff/changie/discussions/237
